example/ddp/ddp.ttl

```diff
@@ -0,0 +1,32 @@
+@prefix core: <https://ontology.okp4.space/core/> .
+@prefix zone: <https://ontology.okp4.space/dataverse/zone/> .
+@prefix owl: <http://www.w3.org/2002/07/owl#> .
+@prefix topic: <https://ontology.okp4.space/thesaurus/topic/> .
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
+
+<https://ontology.okp4.space/dataverse/zone/metadata/86500dd6-080c-4565-8779-8a0aa052f166> a owl:NamedIndividual,  
+        <https://ontology.okp4.space/metadata/zone/GeneralMetadata> ;
+    core:describes zone:d74127ac-d411-4fef-9d65-73c6fc22315d ;  
+    core:hasDescription "Porté par CERFRANCE Bretagne, la data est dans le pré (ou DDP) est un data space dont l'objectif est de valoriser les données agricoles (techniques et comptables) au travers d’un mécanisme de fixation du prix juste pour les produits laitiers, pour ensuite fournir des conseils spécifiques à chaque agriculteur."@fr ;
```

Comment:
I think `data space` should be replaced by `zone`.

Reply:
@ccamel i checked the files, read the test code made several amendment commits and still don't see what still causes the test to fail.

Reply:
@CedrickArmel did you rebased against `main` after #152 merge?